da/blob/verifier.go

```diff
@@ -92,10 +109,6 @@ func (bv *verifier[_, BlobSidecarsT]) verifySidecars(
 		return bv.verifyKZGProofs(sidecars)
 	})
 
-	g.Go(func() error {
-		return sidecars.ValidateBlockRoots()
```

Comment:
q: why was this dropped?

Reply:
The function ensures that all sidecars point to the same beacon block header by the latter's HTR, which is something achieved already (among other guarantees) by the equality test in the loop above.